yacc/reader.c

```diff
@@ -58,10 +58,16 @@ static size_t safe_fwrite(void *ptr, size_t nmemb, FILE *f) {
     return f == NULL ? nmemb : fwrite(ptr, 1, nmemb, f);
 }
 
-static unsigned char caml_ident_start[32] =
-"\000\000\000\000\000\000\000\000\376\377\377\207\376\377\377\007\000\000\000\000\000\000\000\000\377\377\177\377\377\377\177\377";
-static unsigned char caml_ident_body[32] =
-"\000\000\000\000\200\000\377\003\376\377\377\207\376\377\377\007\000\000\000\000\000\000\000\000\377\377\177\377\377\377\177\377";
+static unsigned char caml_ident_start[32] = {
+  '\000', '\000', '\000', '\000', '\000', '\000', '\000', '\000', '\376',
+  '\377', '\377', '\207', '\376', '\377', '\377', '\007', '\000', '\000',
+  '\000', '\000', '\000', '\000', '\000', '\000', '\377', '\377', '\177',
+  '\377', '\377', '\377', '\177', '\377' };
```

Comment:
Nitpick: could you make each line 5 or 8 characters wide? Currently they are 9 characters wide, and I know to count in base 5 or (for arrays of size 32) 8, but not in base 9.

Reply:
Done, aligned to 8.